### PR TITLE
Setup backend routes with encryption/decryption sending

### PR DESCRIPTION
**Pull request by Aeleck**
Edited routes.py and steg-core.py to enable encryption/decryption on the backend. This allows HTTP post requests to be sent to the server and it will respond with an image if using /api/encode and raw text is using /api/decode for now until an agreed upon format is used. 
To test, I ran flask run, and used Postman to setup the body of the request with an image, titled "image", pointed it to the low_res.png image on my computer, and entered a message into the next field, titled "message" and sent the request.
The server responded with an image, which I saved to my local drive.
I then used another equest to /api/decode with the body having the image I just saved as 'image' titled as well. The server responded with the raw decrypted text.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[[source]]
```
Comment (llucas314) [inline Pipfile:1]: Delete the pip files in the root directory. These get generated if you run pipenv outside of the backend folder.

File: backend/app/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-from flask import Flask
+from flask import Flask, request, send_file, Response
```
Comment (llucas314) [inline backend/app/__init__.py:1]: You don't need the extra imports since they are called in other files.

File: backend/flask.env [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export FLASK_APP=steg_api.py
```
Comment (llucas314) [inline backend/flask.env:1]: This should be a local file.

File: backend/steg_api.py [hunk 4]
```diff
@@ -1 +1 @@
-from app import app
+from app import api
```
Comment (llucas314) [inline backend/steg_api.py:1]: Why did you change this? It didn't work until I changed it back. The app is imported here from the app folder.

File: frontend/Pipfile [hunk 5]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (llucas314) [inline frontend/Pipfile:1]: Should be deleted. Same as the other one.

**Comment by Aeleck** [comment]
Cancelling request, other branch created with proper changes added in
